README.md

```diff
@@ -28,7 +28,7 @@ Platform Targets:
  * WebRTC features (VoIP and Video calling) are only available in Chrome & Firefox.
  * Mobile Web is not currently a target platform - instead please use the native
    iOS (https://github.com/matrix-org/matrix-ios-kit) and Android
-   (https://github.com/matrix-org/matrix-android-sdk) SDKs.
+   (https://github.com/vector-im/element-android) SDKs.
```

Comment:
Since we're referencing the SDK layer, I believe we actually want https://github.com/matrix-org/matrix-android-sdk2 (still not quite sure why it's called that).

```suggestion
   (https://github.com/matrix-org/matrix-android-sdk2) SDKs.
```

Reply:
I didn't realize the repo existed because the code lives in the main repo as well, iirc

Reply:
Ah right, looking again, I guess the SDK repos says to send PRs to EA actually... I guess here though, the SDK is slightly more correct, since it's saying what others should use.